### PR TITLE
feat: remove framework support

### PR DESCRIPTION
**Pull request by Weakky**
## TODO

- [x] Remove/update postinstall script

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+      throw err
+    }
+  }
+}
+
+ensureDepIsInstalled('@nexus/schema')
+ensureDepIsInstalled('graphql')
+ensureDepIsInstalled('@prisma/client')
+
+export * from './plugin'
```
Comment (jasonkuhrt) [inline src/index.ts]: Let's make this module export `nexusPrisma`. This should be the function that could be accessed only previously like so:

```ts
import { nexusSchemaPrisma } from 'nexus-plugin-prisma/schema'
```

**Review by jasonkuhrt — COMMENTED** [review]
Let's move the examples to the @nexus/schema repo examples folder. To support centralization strategy. Changes go into this PR https://github.com/graphql-nexus/schema/pull/500.

**Comment by macrozone** [comment]
why is framework support beeing removed? it just got added and we were told to use now framework instead of schema if possible. There is no issue linked that describes the reasoning

File: scripts/build-module-facades.js [hunk 2]
```diff
@@ -23,12 +23,8 @@ const path = require('path')
...
 // prettier-ignore
 const facades = [
-  ['client.d.ts',  "export * from '@prisma/client'"  + os.EOL],
-  ['client.js',    "module.exports = require('@prisma/client')" + os.EOL],
-  ['schema.d.ts',  "export * from './dist/schema-entrypoint'"  + os.EOL],
-  ['schema.js',    "module.exports = require('./dist/schema-entrypoint')" + os.EOL],
-  ['typegen.js',   "module.exports = require('./dist/schema/typegen/static')" + os.EOL],
-  ['typegen.d.ts', "export * from './dist/schema/typegen/static'" + os.EOL],
+  ['schema.d.ts',  "export { nexusPrisma as nexusSchemaPrisma } from './dist'"  + os.EOL],
+  ['schema.js',    "exports.nexusSchemaPrisma = require('./dist').nexusPrisma" + os.EOL],
```
Comment (Weakky) [inline scripts/build-module-facades.js:27]: updated the /schema "facade"

**Review by jasonkuhrt — APPROVED** [review]

**Comment by valentinpalkovic** [comment]
@macrozone See the announcement: https://github.com/prisma-labs/graphql-framework-experiment/issues/1432

**Pull request merged**